.github/workflows/main.yml

```diff
@@ -56,34 +56,3 @@ jobs:
 
       - name: Run tests
```

Comment:
even though we're not publishing here, but should run the build target to make sure everything compiles before merge.

Reply:
Makes sense.